src/views/shared/SecurityBeacon.js

```diff
@@ -64,7 +67,14 @@ define(['okta', 'util/Animations'], function (Okta, Animations) {
         viewport: $('body')
       },
       hide: {event: false, fixed: true},
-      show: {event: false, delay: 200}
+      show: {event: false, delay: 200},
+      events: {
+        move: function(event, api) {
```

Comment:
do we need this event? I would think that with the lines you added at the beginning of this function (48-51) it should all work